iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/topology/ConnectVoltageLevelOnLineTest.java

```diff
@@ -98,6 +100,11 @@ void testSetters() {
     @Test
     void testCompleteBuilder() throws IOException {
         Network network = createNbNetworkWithBusbarSection();
+        BusbarSection bbs = network.getBusbarSection("bbs");
```

Comment:
Why do you need the extension in this case? The result is the same without right?

Reply:
it's used for the test coverage, the result changes a bit (see the changes in fictitious-line-split-vl-complete.xml )